Source/WTF/wtf/WorkQueue.h

```diff
@@ -41,13 +41,13 @@
 
 namespace WTF {
 
-class WorkQueueBase : public FunctionDispatcher, public ThreadSafeRefCounted<WorkQueueBase>, protected ThreadLike {
+class WorkQueueBase : public ThreadSafeRefCounted<WorkQueueBase>, protected ThreadLike {
 public:
     using QOS = Thread::QOS;
 
-    ~WorkQueueBase() override;
+    WTF_EXPORT_PRIVATE virtual ~WorkQueueBase();
 
-    WTF_EXPORT_PRIVATE void dispatch(Function<void()>&&) override;
+    WTF_EXPORT_PRIVATE void dispatchInternal(Function<void()>&&);
```

Comment:
It doesn't sound like dispatchInternal, internal function, should be public.

I think we typically call this dispatchImpl().